test/games/strategy/engine/message/EndPointTest.java

```diff
@@ -6,10 +6,9 @@
 import junit.framework.TestCase;
 
 public class EndPointTest extends TestCase {
-  @SuppressWarnings("rawtypes")
   public void testEndPoint() {
     final EndPoint endPoint = new EndPoint("", Comparator.class, false);
-    endPoint.addImplementor(new Comparator() {
+    endPoint.addImplementor(new Comparator<Object>() {
```

Comment:
It was `EndPoint` btw... Problematic since Endpoint also exists ^^
But it needs to be Object for whatever reason... otherwise the test fails... Compilation is no problem... even using the false type caused no errors